models/users.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../db/connection');
+
+exports.selectUsers = () => {
+	return db.query('SELECT * FROM users').then((result) => {
+		const users = result.rows.map((user) => {
```

Comment:
No need to map! You need all the properties you get. 😄

Reply:
I thought I only need the key of 'username'...!🥲

Reply:
Yes you are right! Sorry just saw this.